etc/ci/scenario/servo_test_frame_rate_base_page.py

```diff
@@ -0,0 +1,71 @@
+import sys
+import time
+import subprocess
+import common_function_for_servo_test
+from selenium.webdriver.common.by import By
+
+
+def operator():
+    # Step 1. Open mossel
+    cmd_start_servo = [
+        f'hdc shell aa force-stop org.servo.servo',
```

Comment:
you will need to split the command into an array, like `['hdc', 'shell', 'aa force-stop org.servo.servo'], otherwise it will not run. Maybe it works on windows, but on Linux and macos you can't use one string for the whole command. Everything after hdc shell can be one string again - hdc will pass it to the shell on the phone, so it doesn't matter for the rest.